sms_marketing/SMS_mkt_build_send-and_get_reports_from_a_SMS_mailing.rst

```diff
@@ -21,57 +20,81 @@ Go to :menuselection:`SMS Marketing --> Create`.
 .. image:: ./media/sms_marketing1.png
    :align: center
 
-First, I’ll be able to specify a *subject* name that will help me to remember what the mailing is about.
-Under *recipients*, I’ll choose to whom I’d like the SMS to be sent. If I choose *Contact*, all my Odoo contacts (vendor, customers, etc.) will receive my SMS.
-From there, I can refine my recipient list with the *add filter* option. In the example below, I choose to send it to my contacts living in Belgium.
+| First, you will be able to specify a *subject* name that will help you to remember what the
+  mailing is about.
+
```

Comment:
You should either:
- use blank lines in RST to create a new paragraph in HTML
or
- use the `|` notation to force a line break within a paragraph

but not both at the same time.